### PR TITLE
Lrowac2pds App and Test Conversion

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Description

App conversion for lrowac2pds and test conversion to gtests

## Motivation and Context

Testing Sprint

## How Has This Been Tested?

Added 4 gtests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/lro/apps/lrowac2pds/lrowac2pds.cpp [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+#include "History.h"
+#include "Stretch.h"
+#include "iTime.h"
+#include "ProcessExport.h"
+#include "PvlToPvlTranslationManager.h"
+#include "PvlFormatPds.h"
+#include "md5.h"
+#include "md5wrapper.h"
+#include "OriginalLabel.h"
+#include <sstream>
```
Comment (ladoramkershner) [inline isis/src/lro/apps/lrowac2pds/lrowac2pds.cpp:26]: This is padentic and I realize this is how other files are structured as well, but why not place these includes in the header file?

Comment (amystamile-usgs) [reply isis/src/lro/apps/lrowac2pds/lrowac2pds.cpp:26]: That is a good question that I don't have the answer for. I have just been following the instructions in the wiki for writing tests in which we copy everything over from the app's main.cpp and stick to the header [template](https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#applicationh-template) provided. So I am not sure as to the reason why we are doing it this way.

Comment (ladoramkershner) [reply isis/src/lro/apps/lrowac2pds/lrowac2pds.cpp:26]: Thank you for pointing me towards that documentation. I did not get the chance to work on this project so I was not aware of it! I think it is totally fine the way it is, I was just curious if there was a specific reason for this choice (probably was, but it is lost to time)

**Comment by ladoramkershner** [comment]
The conversion looks good to me!

**Review by ladoramkershner — APPROVED** [review]

**Pull request merged**